docs/build-smart-contracts.md

```diff
@@ -8,11 +8,13 @@ The Polkadot relay chain will not support smart contracts natively.  However, pa
 
 ## Resources
 
-Here are the list of current resources available to developer who want to get started writing smart contracts to deploy on parachains based on Substrate.
+Here is the list of current resources available to developers who want to get started writing smart contracts to deploy on parachains based on Substrate.
+
+- [ink!](https://github.com/paritytech/ink) - Parity's ink! eDSL to write smart contracts.
+- [Deploying your first contract](https://github.com/paritytech/ink/wiki/Deploying-Your-First-Contract) - Guide to deploy the provided `flipper` contract.
```

Comment:
This was covered in a different PR as well, but lines 14 - 15 are out of date and should be removed.